src/main/java/com/bobocode/nasapicturesstealer/persistance/entity/Camera.java

```diff
@@ -0,0 +1,40 @@
+package com.bobocode.nasapicturesstealer.persistance.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+import static jakarta.persistence.GenerationType.IDENTITY;
+
+@Builder
+@Data
+@AllArgsConstructor
```

Comment:
Remove @AllArgsConstructor. Id should be specified only by otm framework

Reply:
Done